doc/api/errors.md

```diff
@@ -1655,6 +1655,12 @@ recommended to use 2048 bits or larger for stronger security.
 A TLS/SSL handshake timed out. In this case, the server must also abort the
 connection.
 
+<a id="ERR_TLS_INVALID_PROTOCOL_METHOD"></a>
+### ERR_TLS_INVALID_PROTOCOL_METHOD
+
+An invalid `secureProtocol` method was specified. It was either unknown, or
```

Comment:
```suggestion
The specified  `secureProtocol` method is invalid. It is  either unknown, or
```

Reply:
☝️ Just a suggestion. Ignore it if you don't like it. I think it's simpler and more direct, but whatevs.